stubs/redis/redis/cluster.pyi

```diff
@@ -215,23 +217,27 @@ class ClusterPipeline(RedisCluster[_StrType], Generic[_StrType]):
     def script_load_for_pipeline(self, *args, **kwargs) -> None: ...
     def delete(self, *names): ...
 
-def block_pipeline_command(name: str) -> Callable[..., Any]: ...
+def block_pipeline_command(name: str) -> Callable[..., NoReturn]: ...
 
 class PipelineCommand:
-    args: Any
-    options: Any
-    position: Any
-    result: Any
-    node: Any
+    args: Sequence[EncodableT]
+    options: _ParseResponseOptions
+    position: int | None
+    result: Any | None
+    node: Any | None
     asking: bool
-    def __init__(self, args, options: Any | None = ..., position: Any | None = ...) -> None: ...
+    def __init__(self, args: Sequence[EncodableT], options: Any | None = ..., position: int | None = ...) -> None: ...
+
+_ParseResponseCallback: TypeAlias = Callable[[Connection, EncodableT, _ParseResponseOptions], Any]
```

Comment:
This implies that the callback has to take a dictionary as its third parameter. But that doesn't look right. It looks to me like the callback takes a variadic number of keyword-only arguments: https://github.com/redis/redis-py/blob/b5ebada842c715e9ee74ea638a7e6d11afcddae1/redis/cluster.py#L2269